fingertip/plugins/backend/qemu.py

```diff
@@ -154,27 +170,36 @@ def run(self, load=SNAPSHOT_BASE_NAME, extra_args=[]):
         run_args += self.usernet._netspec_cmd()
 
         self.image = os.path.join(self.vm.path, 'image.qcow2')
-        cloned_to_tmp = False
-        if self._image_to_clone:
-            # let's try to use /tmp (which is, hopefully, tmpfs) for transients
-            # if it looks empty enough
-            required_space = os.path.getsize(self._image_to_clone)
-            if self.vm._transient:
-                # Would be ideal to have it global (and multiuser-ok)
-                tmp_free_lock = path.cache('.tmp-free-space-check-lock')
-                with fasteners.process_lock.InterProcessLock(tmp_free_lock):
-                    if temp.has_space(required_space, where='/tmp',
-                                      safety_constant='4G', target_free=.5):
-                        self.image = temp.disappearing_file(
-                            '/tmp', hint='qemu'
-                        )
-                        self.vm.log.info('preloading image to /tmp...')
-                        reflink.auto(self._image_to_clone, self.image)
-                        self.vm.log.info('preloading image to /tmp completed')
-                        cloned_to_tmp = True
-            if not cloned_to_tmp:
-                reflink.auto(self._image_to_clone, self.image)
+        self.efivars = os.path.join(self.vm.path, 'efivars.qcow2')
+        cloned_to_tmp = set()
+        if self._image_to_clone or self._efivars_to_clone:
+            pairs = []
+            if self._image_to_clone:
+                pairs.append((self._image_to_clone, self.image))
+            if self._efivars_to_clone:
+                pairs.append((self._efivars_to_clone, self.efivars))
+
+            for image_to_clone, image in pairs:
```

Comment:
rolled back the changes to the /tmp offloading logic in a fixup commit for the reasons above + because the loop assigned to self.image irrespective of what it iterated over, disk or efivars